### PR TITLE
Task/first

### PR DESCRIPTION
**Pull request by Parker-ygfert**

### CODE REVIEW

File: app/models/shipment.rb [hunk 1]
```diff
@@ -1,4 +1,10 @@
 class Shipment < ApplicationRecord
   belongs_to :company
-  belongs_to :shipment_item
+  has_many :shipment_items
+
+  def group_shipment_items
+    shipment_items.group(:description).order('count_all desc').count.map do |item|
+      {description: item[0], count: item[1]}
```
Comment (alChaCC) [inline app/models/shipment.rb]: heyhey ~ how to make it more readable ?

Comment (alChaCC) [reply app/models/shipment.rb]: hint: `item[0]` , `item[1]` and `map { |item| ...`

File: spec/models/shipment_spec.rb [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
-  # write your tests here
+  let(:company) { Company.create(name: 'New Co') }
+  subject { described_class.create(company: company) }
+
+  before do
+    subject.shipment_items.create(
+      description: 'Apple Watch',
+      shipment: subject
+    )
+    2.times do
```
Comment (alChaCC) [inline spec/models/shipment_spec.rb]: can try to use `create_list`

File: app/models/shipment.rb (hunk 1 above)
Comment (Parker-ygfert) [reply app/models/shipment.rb]: can I refactor like this?
```ruby
def group_shipment_items
  shipment_items.group(:description).order('count_all desc').count
end

def set_shipment_items
  group_shipment_items.map do |item|
    {description: item[0], count: item[1]}
  end
end
```

Comment (alChaCC) [reply app/models/shipment.rb]: hey hey ~ nope~
`item[0]` and `item[1]` is not readable~~~ you can try to find another way to show it ~~

hint try to change `|item|`

**Comment by alChaCC** [comment]
@Parker-ygfert sorry I can not assign to you ~ anyways finished task 1 review!

File: app/models/shipment.rb (hunk 1 above)
Comment (Parker-ygfert) [reply app/models/shipment.rb]:
```ruby
def set_shipment_items
  group_shipment_items.map do |item, count|
    {description: item, count: count}
  end
end
```

got it!
I should use `key` and `value`
cause the method `group_shipment_items` returns a `Hash`

**Comment by Parker-ygfert** [comment]
No worries, and thanks to your review :)

**Comment by Parker-ygfert** [comment]
Hi, @alChaCC 
I finished the refactoring of shipment model method and rspec

**Pull request closed without merging**